devel/olm/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+name                olm
+version             2.3.0
+categories          devel security
+platforms           darwin
+maintainers         {@scarface-one disroot.org:scarface} openmaintainer
+description         An implementation of the Double Ratchet cryptographic ratchet in C++
+long_description    An implementation of the Double Ratchet cryptographic ratchet described by \
+                    https://whispersystems.org/docs/specifications/doubleratchet/, written in C \
+                    and C++11 and exposed as a C API.
+
+license             Apache-2
+homepage            http://git.matrix.org/git/olm/about/
+
+master_sites        http://git.matrix.org/git/olm/snapshot
+
+checksums           rmd160  f848e0fe13866943c0af07a2cbbfc16aee6de229 \
+                    sha256  533714fb84860e04c185790d16ef9085f15e902c2105db941d5c7e92b0565ef8 \
+                    size    480801
+
+use_configure       no
+
+build.cmd           make
+build.pre_args-delete all
+build.pre_args-append static
```

Comment:
`build.cmd` defaults to `make`. And you can use `build.target static` here. See https://guide.macports.org/#reference.phases.build for details.

Reply:
Is it possible to build and use a shared library? Building a static library means that for every update, dependents need to be rev-bumped. This is easy to forget without a comment.